systemtests/src/main/java/io/enmasse/systemtest/DockerCmdClient.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2018, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.systemtest;
+
+import io.enmasse.systemtest.executor.ExecutionResultData;
+import io.enmasse.systemtest.executor.Executor;
+import org.slf4j.Logger;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Class represent abstract client which keeps common features of client
+ */
+public class DockerCmdClient {
+    private static Logger log = CustomLogger.getLogger();
+    private static final Object lock = new Object();
+    private static int DEFAULT_SYNC_TIMEOUT = 120000;
+    private static String DOCKER_COMMAND;
+
+    public static ExecutionResultData pull(String name, String repository) {
```

Comment:
just minor note that I would expected 'repository' as first parameter (as it is used in command bellow) only my personal opinion